app/models/related-identifier.js

```diff
@@ -0,0 +1,69 @@
+import DS from 'ember-data';
+import Fragment from 'ember-data-model-fragments/fragment';
+import { validator, buildValidations } from 'ember-cp-validations';
+import { computed } from '@ember/object';
+
+const Validations = buildValidations({
+  schemeUri: [
+    validator('url-format', {
+      allowBlank: true,
+      require_tld: false,
+      message: 'Please enter a valid URL.',
+    }),
+  ],
+  relatedIdentifier: [
+    validator('url-format', {
```

Comment:
Consider writing a generic validator, using the `validator` library. Does not have to cover all relatedIdentifierType.